tests/accounts/test_csrf.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from django.test import Client
+from django.urls import reverse
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def _add_installed_apps(settings):
```

Comment:
this fixture does not add/inject any additional apps

furthermore, settings that should be common to all projects belong in `testapp/settings.py` - in this case you don't need a fixture, but add the settings to `testapp/settings.py`